### DISCUSSION TITLE
Relational data is added but not deleted in MySQL.

### DISCUSSION THREAD

**Discussion by yeolwoosung** [Budibase/budibase#6923] [Help] [answered]
Dear everyone.

There's a problem.
I am using mysql and relational data is added but not deleted.

See the gif file below.
1. There are products mentioned in the BRAND table.
2. Exclude some data out of all 50 through row editing.
3. After saving, the correct message will appear.
4. But when I check again, it is still 50.
5. And when I add the 2 data back, it works fine.

That is, you can add, but not delete.

I don't know if this is a bug or if I'm using it wrong.
Help.

![13](https://cdn.wise-fashion.com/tmp/20220727/EDr67cptizgI8JK034Pv.gif)

**Comment by melohagan** [comment, marked as answer]
Hi @SungYeolWoo 

Thank you for bringing this to our attention, and for the helpful GIF!

I have been able to reproduce the issue and have raised a bug here: https://github.com/Budibase/budibase/issues/6980
